plugins/modules/web_infrastructure/jenkins_build.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/python
+#
+# Copyright: (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: jenkins_build
+short_description: Manage jenkins builds
+version_added: 1.1.0
+description:
+    - Manage Jenkins builds with Jenkins REST API.
+requirements:
+  - "python-jenkins >= 0.4.12"
+author: "Brett Milford (@brettmilford)"
+options:
+  args:
+    description:
+      - A list of parameters to pass to the build.
+    required: false
+    type: dict
+  name:
+    description:
+      - Name of the Jenkins job to build.
+    required: true
+    type: str
+  build_number:
+    description:
+      - An integer which specifies a build of a job. Is required to remove a build from the queue.
+    required: false
+    type: int
+  password:
+    description:
+      - Password to authenticate with the Jenkins server.
+    required: false
+    type: str
+  state:
+    description:
+      - Attribute that specifies if the build is to be created or deleted.
+    required: false
+    default: present
+    choices: ['present', 'absent']
+    type: str
+  token:
+    description:
+      - API token used to authenticate with the Jenkins server.
+    required: false
+    type: str
+  url:
+    description:
+      - URL of the Jenkins server.
+    required: false
+    default: http://localhost:8080
+    type: str
+  user:
+    description:
+       - User to authenticate with the Jenkins server.
+    required: false
+    type: str
+'''
+
+EXAMPLES = '''
+- name: Create a jenkins build using basic authentication
+  community.general.jenkins_build:
+    name: "test-check"
+    args:
+      cloud: "test"
+      availability_zone: "test_az"
+    user: admin
+    password: asdfg
+    url: http://localhost:8080
+'''
+
+RETURN = '''
+---
+name:
+  description: Name of the jenkins job.
+  returned: success
+  type: str
+  sample: "test-job"
+state:
+  description: State of the jenkins job.
+  returned: success
+  type: str
+  sample: present
+user:
+  description: User used for authentication.
+  returned: success
+  type: str
+  sample: admin
+url:
+  description: Url to connect to the Jenkins server.
+  returned: success
+  type: str
+  sample: https://jenkins.mydomain.com
+build_info:
+  description: Build info of the jenkins job.
+  returned: success
+  type: dict
+'''
```

Comment:
```suggestion
  type: dict
  sample: ...
'''
```

Reply:
I'd like to supply a sample for this but that's no longer feasible for me unfortunately.